src/config.ts

```diff
@@ -4,12 +4,18 @@ import { ResolvedTraceConfig, Trigger } from './types.js'
 const configSymbol = Symbol('Otel Workers Tracing Configuration')
 
 export type Initialiser = (env: Record<string, unknown>, trigger: Trigger) => ResolvedTraceConfig
+let fallbackConfig: ResolvedTraceConfig
 
 export function setConfig(config: ResolvedTraceConfig, ctx = context.active()) {
+	// I could not get the context to work properly, so I'm using a global fallback.
+	// I suspect this is because I am not initialising the config properly in the WS instrumentation.
+	if (!fallbackConfig) {
```

Comment:
this is actually a bug in this library and/or with context combined with websockets. I tried instrumenting WS on my own and I get the same thing as here: https://github.com/evanderkoogh/otel-cf-workers/issues/14

Reply:
Hey I think it's an async local storage issue. Happy to jump on a call and work on this together if you would like ☺️